### DISCUSSION TITLE
how to display the flowchart with too much nodes

### DISCUSSION THREAD

**Discussion by echoroot8127** [mermaid-js/mermaid#5685] [Q&A]
i'm beginner in mermaid.
Now i met a problem  in practice , just like this code.

```
%%{init*}%%
flowchart LR
id0-->id1-->id2-->id3-->id4-->id5-->id6-->id7-->id10-->id11-->id12-->id13-->id14-->id15-->id16-->id17-->id20-->id21-->id22-->id23-->id24-->id25-->id26-->id27
```

there're too much node up to 27, so the display will like this .
![1](https://github.com/user-attachments/assets/b6b25c5e-8b6d-4e8c-8df1-0a75d5559abf)

it's difficult to view or use it,  i hope the output file like the picture 
![未命名绘图](https://github.com/user-attachments/assets/603ba213-bf8e-4c52-842e-5439231bdf8c)

.how  can i Modify the code,
thank you .

**Comment by movingindex** [comment]
Yes, it will be useful to have the ability to wrap around the flow.  It could still be always left to right, like this:
(reference: https://1c.com.vn/en/news/quy-tac-ve-flowchart)
![Flowchart multiple lines](https://github.com/user-attachments/assets/67bb04be-d216-4639-a6cd-51e99f4560db)
